src/BatchReader/ServiceConfiguration.ts

```diff
@@ -1,3 +1,5 @@
-export interface ServiceConfiguration {
+import { ExchangeConfiguration } from './ExchangeConfiguration'
+
+export interface ServiceConfiguration extends ExchangeConfiguration {
```

Comment:
This `extends ExchangeConfiguration` is unnecessary I think. Aren't we using the ExchangeConfiguration directly?